app/models/katello/events/import_host_errata.rb

```diff
@@ -0,0 +1,17 @@
+module Katello
+  module Events
+    class ImportHostErrata < Katello::Model
+      EVENT_TYPE = 'import_host_errata'.freeze
+
+      def initialize(object_id)
+        @host = ::Host.find(object_id)
```

Comment:
I lean towards handling an invalid host ID properly here so as to produce a better error than the full traceback that would normally occur?